### PR TITLE
feat: connected ec2 server with Ollama and S3

### PR DESCRIPTION
**Pull request by aaronchan73**
- retrieves resume data from S3
- sends resume prompt to Ollama (assuming container is running on port 11434)
- definitely need a better prompt in the future lol

### CODE REVIEW

File: ec2/ec2-docker-setup-commands.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+docker pull ollama/ollama
+docker run -d --name ollama -p 11434:11434 ollama/ollama
+docker exec -it ollama ollama run llama3.2:1b
+curl http://localhost:11434/api/generate -d '{"model":"llama3.2:1b","prompt":"Why is the sky blue?", "stream": false}'
+
+docker start d2a556f0fab2
```
Comment (aaronchan73) [inline ec2/ec2-docker-setup-commands.sh:6]: wouldnt this container id be different for every build? are we using this script or just leaving this as reference

File: ec2/server.py [hunk 2]
```diff
@@ -1,29 +1,49 @@
...
+    aws_session_token=os.getenv("AWS_SESSION_TOKEN"),
+)
+
 
 def load_data(bucket_name, file_name):
     obj = s3.get_object(Bucket=bucket_name, Key=file_name)
-    data = obj['Body'].read()
-    return pickle.loads(data, encoding='bytes')
+    data = obj["Body"].read()
+    return data.decode("utf-8")
```
Comment (meleongg) [inline ec2/server.py:23]: have we tested if this successfully reads input from an S3 bucket?

Comment (aaronchan73) [reply ec2/server.py:23]: Yessir it works!

Comment (meleongg) [reply ec2/server.py:23]: lesgooo

**Pull request merged**